client/settings/fraud-protection/advanced-settings/cards/avs-mismatch.js

```diff
@@ -13,38 +13,48 @@ import FraudProtectionRuleCard from '../rule-card';
 import FraudProtectionRuleDescription from '../rule-description';
 import FraudProtectionRuleCardNotice from '../rule-card-notice';
 
-const AVSMismatchRuleCard = () => (
-	<FraudProtectionRuleCard
-		title={ __( 'AVS Mismatch', 'woocommerce-payments' ) }
-		description={ __(
-			'This filter compares the street number and the post code submitted by the customer against the data on ' +
-				'file with the card issuer.',
-			'woocommerce-payments'
-		) }
-	>
-		<FraudProtectionRuleDescription>
-			{ __(
-				'Buyers who can provide the street number and post code on file with the issuing bank ' +
-					'are more likely to be the actual account holder. AVS matches, however, are not a guarantee.',
+const AVSMismatchRuleCard = () => {
+	const declineOnAVSFailure =
```

Comment:
Showing different messages according to the platform setting for AVS checks according to the design on [Figma](https://www.figma.com/file/WIM5w2Y10SCNBFdMLSVftj/%5BWCPay%5D-Fraud-%26-Risk-Tools?node-id=2930-263372&t=V91lulvTyXqJtCTK-0)